ikasaneip/test/src/main/java/org/ikasan/testharness/flow/rule/IkasanFlowTestRule.java

```diff
@@ -397,9 +397,11 @@ public void fireScheduledConsumer()
         {
             JobDetail jobDetail = ((ScheduledComponent<JobDetail>)consumer).getJobDetail();
             Trigger trigger = newTrigger().withIdentity("name", "group").forJob(jobDetail).build();
+            // adding a sleep in test to give quartz a chance to see a new scheduled time in consumer.scheduleAsEagerTrigger
```

Comment:
This doesnt seem like a solution - adding sleeps are the causes of much of our intermittent failures. Would prefer we tracked the real root of the problem down